documents/operating.md

```diff
@@ -21,10 +21,100 @@ SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
 SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
 ---------------------------------------------------------------------
-- [ ] Types of involvement
-- [ ] Roles and responsibilities
-- [ ] Adding partners
-- [ ] Removing partners
+- [x] Types of involvement
+- [x] Roles and responsibilities
+- [x] Adding partners
+- [x] Removing partners
+
+### Types of Involvement
+1. **Corporate Overlord:** Corporate Overlords of BIFFUD own part of the company.
+They have votes during plotting sessions and can review pull requests to `/corporate`.
+
+2. **Members:** members can have ownership over BIFFUD projects. All Corporate Overlords
+are also members.  Members are entitled to attend BIFFUD meetings.
+
+3. **Minions:** minions are contributing participants in BIFFUD projects. They
+might contribute via code, cash, or expertise.  They are entitled to attend BIFFUD
+meetings.
+
+
+### Corporate Overlord Responsibilities
+Bad Idea Factory is a Limited Liability Corporation, which means that
+partnership can't ~just~ be about the spirit or the lulz -- partners
+have legal responsibilities, ownership, and control over the company
+direction.
+
+Corporate Overlords are expected to...
+
+1. Make a capital contribution upon joining.
+2. Participate and vote in monthly plotting sessions.
+3. Participate in reviews for changes to corporate GitHub.
+4. Contribute original poetry on corporate GitHub.
+5. Stay involved and aware of the projects being undertaken by members.
+6. Help maintain and further the BIFFUD public footprint.
+7. Try to attend BIFFUD events.
+
```

Comment:
Any need here to have sections for Member Responsibilities or Minion Responsibilities?